roofit/roofitcore/inc/RooAddPdf.h

```diff
@@ -32,28 +33,28 @@
 class RooAddPdf : public RooAbsPdf {
 public:
 
-  RooAddPdf() ;
+  RooAddPdf() { TRACE_CREATE }
   RooAddPdf(const char *name, const char *title=0);
   RooAddPdf(const char *name, const char *title,
 	    RooAbsPdf& pdf1, RooAbsPdf& pdf2, RooAbsReal& coef1) ;
   RooAddPdf(const char *name, const char *title, const RooArgList& pdfList) ;
-  RooAddPdf(const char *name, const char *title, const RooArgList& pdfList, const RooArgList& coefList, Bool_t recursiveFraction=kFALSE) ;
+  RooAddPdf(const char *name, const char *title, const RooArgList& pdfList, const RooArgList& coefList, Bool_t recursiveFraction=false) ;
   
   RooAddPdf(const RooAddPdf& other, const char* name=0) ;
   virtual TObject* clone(const char* newname) const { return new RooAddPdf(*this,newname) ; }
-  virtual ~RooAddPdf() ;
+  virtual ~RooAddPdf() { TRACE_DESTROY }
 
   virtual Bool_t checkObservables(const RooArgSet* nset) const ;	
 
   virtual Bool_t forceAnalyticalInt(const RooAbsArg& /*dep*/) const { 
     // Force RooRealIntegral to offer all observables for internal integration
```

Comment:
When you find this kind of old-style docs, move them above the function and replace `// --> ///` for doxygen to find it.

Reply:
Okay!